atc/db/migration/migrations/1594926103_add_access_token_and_claims.down.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    DROP TABLE IF EXISTS access_tokens;
```

Comment:
I couldn't remember if this worked on Postgres 9 so I deployed and then ran `migrate --migrate-db-to-version=1591737494` and it worked fine.

I remember now that it's `DROP COLUMN IF EXISTS` that only exists in certain versions of Postgres.